### PR TITLE
Loading state for editing artwork/buyer/artist profile

### PR DESCRIPTION
**Pull request by emilyzhong**
## Loading state for editing artwork/buyer/artist profile
[Asana Ticket](https://app.asana.com/0/860297512787618/1110722831522308/f)
* Added form loading overlay upon form submission
* Added `fullPage` prop to `LoadingOverlay` for full page overlays vs local (ie within the form)
* Added `LoadingOverlay` to more full page locations

### Related PRs
Loading Bars: https://github.com/calblueprint/SFAI/pull/59 

### Migrations
None

### Tests Performed, Edge Cases
None

### Screenshots
screen shot 2019-02-26 at 10 41 06 am

CC: @by-co

### CODE REVIEW

File: app/javascript/components/receipts/TransactionForm.jsx [hunk 1]
```diff
@@ -152,9 +153,7 @@ class TransactionForm extends React.Component {
 
   render() {
     if (!this.state.didMount) {
-      return (
-        <div><h2>Loading</h2></div>
-      );
+      return <LoadingOverlay fullPage={true} />;
```
[the remark this answers is not in the document]
Comment (emilyzhong) [reply app/javascript/components/receipts/TransactionForm.jsx]: Should I include a loading overlay for the form for when it gets submitted?

[the remark this answers is not in the document]
Comment (by-co) [reply app/javascript/components/receipts/TransactionForm.jsx]: Yes that would be good :)
